benchmarks/nemo/nemolite2d/kernels/c_family/boundary_conditions_kern.c

```diff
@@ -170,7 +170,12 @@ __kernel void bc_ssh_code(int width,
   if(ji==0 || ji > (width-2))return;
   if(jj==0 || jj > (nrow-2))return;
 #else
-inline void bc_ssh_code(int ji, int jj, int width,
+#if defined(KOKKOS_INLINE_FUNCTION)
+KOKKOS_INLINE_FUNCTION
+#else
+inline
```

Comment:
Presumably a new-line in C code acts as a separator (as there's no trailing whitespace after "inline"? I tried googling but didn't have much luck.

Reply:
I assume this is a performance optimisation?

Reply:
Yes it acts as separator, I tested all the manual_version that re-use the same kernels and they still work.

> I assume this is a performance optimisation?

It seems when using a host device it just adds the `inline` modifier as well, so it is not improving the performance at the moment, but it is the recommended way in kokkos to prefix functions so I decided to put it.

When running on other devices like CUDA, it adds the `__device` modifier (for instance if one tries to compile the rawpointers version with CUDA, the compilation will succeed).